lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,112 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # prevent autorun from running tests when the vm closes
+  def self.autorun
+  end
+end
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      # represents the data associated with a test
+      class TestCase
+        include Concord.new(:class_name, :test_method)
+        include Adamantium
+
+        attr_reader :class_name
+
+        def run_passes?(reporter)
+          ::Minitest::Runnable.run_one_method(klass, test_method, reporter)
+          reporter.passed?
+        end
+
+        def method_name
+          test_method[5..-1]
+        end
+        memoize :method_name
+
+        private
+
+        def klass
+          Mutant.constant_lookup(class_name)
+        end
+        memoize :klass
+      end
+
+      register 'minitest'
+
+      def initialize
+        @output = StringIO.new
+        @reporter = make_reporter(@output)
+      end
+
+      def setup
+        $LOAD_PATH << 'test' unless $LOAD_PATH.include?('test')
```

Comment:
I guess that's slightly better since it removes duplication.

Is there any harm in doing `$LOAD_PATH.concat(%w[test lib])`? If those directories *do* happen to exist in the load path, appending them at the end isn't going to make a difference.